lib/image-inspector-client/version.rb

```diff
@@ -1,4 +1,4 @@
 # Client for image inspector: https://github.com/openshift/image-inspector
 module ImageInspectorClient
-  VERSION = '1.0.3'
+  VERSION = '1.0.3'.freeze
```

Comment:
@cben  Do we intend to bump this in another PR?

Reply:
yes.
Well I opened this PR from Mooli's branch, which felt clever but doesnt allow me to do any changes :-)